#TODO: Define what your custom message includes here
#Here's a list of built-in primitive types: http://wiki.ros.org/msg
float32 direction0
float32 direction1
float32 direction2
float32 direction3